### PR TITLE
8.3.2.0

### PR DESCRIPTION
**Pull request by arnaumartifont**
Checking Chartboost minimum iOS version available to prevent making calls that would expect a callback.

### CODE REVIEW

File: Chartboost/ChartboostRouter.m [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 static NSString * const kChartboostAppIdKey        = @"appId";
 static NSString * const kChartboostAppSignatureKey = @"appSignature";
+static NSString * const kGADMAdapterMinimumOSVersion = @"10.0";
```
Comment (daniel-barros) [inline Chartboost/ChartboostRouter.m]: Should we use a different name for MoPub? GADM Are the GoogleMobileAds prefix.
kChartboostMinimumOSVersion?

File: Chartboost/ChartboostRouter.m [hunk 2]
```diff
@@ -61,6 +62,12 @@ + (void)setDataUseConsentWithMopubConfiguration
 
 + (void)startWithParameters:(NSDictionary *)parameters completion:(void (^)(BOOL))completion
 {
+    if (SYSTEM_VERSION_LESS_THAN(kGADMAdapterMinimumOSVersion)) {
+        NSLog(@"Chartboost minimum supported OS version is iOS %@. Requested action is a no-op.", kGADMAdapterMinimumOSVersion);
```
Comment (daniel-barros) [inline Chartboost/ChartboostRouter.m]: I don't see any NSLog in the project. I think it's better to stick to MPLogEvent to track issues, which is used everywhere.
We probably should define a new error in the NSError+ChartboostErrors.h for this

**Review by daniel-barros — CHANGES_REQUESTED** [review]
Some changes didn't follow what is done in the rest of the code

File: Chartboost/ChartboostRouter.m (hunk 1 above)
Comment (arnaumartifont) [reply Chartboost/ChartboostRouter.m]: 👍

**Review by daniel-barros — APPROVED** [review]

**Pull request closed without merging**